python/dgl/sparse.py

```diff
@@ -53,6 +55,26 @@ def infer_broadcast_shape(op, shp1, shp2):
     rst = tuple(max(d1, d2) for d1, d2 in zip(pad_shp1, pad_shp2))
     return rst[:-1] + (1,) if op == "dot" else rst
 
+def use_libxsmm(flag):
+    r"""Set whether Dgl use libxsmm for cpu Spmm operation at runtime.
```

Comment:
* Use all capitalized `DGL`.
* Give a pointer to the libxsmm official repo or doc.
* `cpu` -> `CPU`
* `Spmm` needs explanation. To omit such burden, I suggest just say `Set whether DGL uses libxsmm at runtime.`

Reply:
Make sense, use 'Set whether DGL uses libxsmm at runtime' instead and add the link to official repo of libxsmm.